src/extension.ts

```diff
@@ -239,6 +250,8 @@ function launchMetals(
     serverOptions,
     clientOptions
   );
+  const features = new MetalsFeatures();
+  client.registerFeature(features);
 
   function registerCommand(command: string, callback: (...args: any[]) => any) {
```

Comment:
I just remembered that we already have `client-commands.ts` where we define the list of client commands, that are then automatically registered.

If possible, it would be nice to centralize the tvp commands as well.

Reply:
I made a quick attempt to use that but failed somehow. Don't remember the details. I agree it would be good to consolidate the logic for registering commands.